Dockerfile

```diff
@@ -17,6 +17,12 @@ RUN apt-get -qq update &&\
     rsync \
     netcat \
     psmisc \
+    bc \
+    groff \
```

Comment:
Can you test the runner without groff ? If the aws cli works without groff, the image size will be much smaller.

If it doesn't work, we will add groff

Reply:
You're right, groff is not needed, I don't know why it's added in the gitlab-org base image.